CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## Unreleased
+
+* [Enhancement] Drop support for Python 3.8; Update the Ubuntu base image to
```

Comment:
I understand I am nitpicking, but a semicolon does not trigger capitalisation of the next word.

Thus, either use:

```suggestion
* [Enhancement] Drop support for Python 3.8. Update the Ubuntu base image to
```

... or else, use:

```suggestion
* [Enhancement] Drop support for Python 3.8; update the Ubuntu base image to
```